.mergify.yml

```diff
@@ -12,13 +12,17 @@ pull_request_rules:
       merge:
         method: merge
         strict: true
-  - name: automatic merge when CI passes and 1 approving reviews
+  - name: automatic merge when CI passes and all requested reviews approved
```

Comment:
@daisuke-yoshimoto I would remove the changes in this file.. we are trying to align all repos to look the same. Is there any reason why you changed the defaults?